# Remote Management Request
temoto_2/RMPRequest rmp

# Field that contains the speech specifiers
temoto_2/SpeechSpecifier[] speech_specifiers

---

# Remote Management Response
temoto_2/RMPResponse rmp

# name of the package
string package_name

# name of the launch/run file
string executable

# return the name of the topic where the task should subscribe to
string topic
